rebar.config

```diff
@@ -31,3 +31,10 @@
     {plt_apps, all_deps}
    ]
 }.
+
+{profiles, [
+    {test, [
+        {deps, [{meck, {git, "https://github.com/eproxus/meck.git", {ref, "dfb47c544bdf14b5c26ff2651cf9339dbbe2f50e"}}},
```

Comment:
why pin a commit here ?

Reply:
There is no tag for the latest version and I think a commit works as well as a particular version.

Reply:
Changed to use the hex package with the latest tagged version.